network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -43,13 +43,23 @@ <h1 class="h1-heading {% if uses_menu %}d-none d-md-block{% endif %}">
         </div>
       {% endblock %}
         {% block html_content %}
-          {% for block in page.body %}
-            {% if block.block_type == 'heading' %}
-                <h1>{{ block.value }}</h1>
-            {% else %}
-                {% include_block block %}
-            {% endif %}
-          {% endfor %}
+            {% for block in page.body %}
```

Comment:
I did not seem to find a specific block template for paragraph, so what I am doing here is saying if the block is a paragraph, open its own container and row, and then following the logic for heading from the original template, then finally closing the container. 

If it is not a paragraph, then render the block template, which in its own template will open a new container